test/mocks/withMockRouter.js

```diff
@@ -24,6 +24,10 @@ export const withMockRouterContext = mockRouter => {
     }
   }
 
+  MockRouterContext.propTypes = {
+    children: PropTypes.element.isRequired,
```

Comment:
When should we use `element`instead of `node`?

Reply:
// Anything that can be rendered: numbers, strings, elements or an array
// (or fragment) containing these types.
  optionalNode: PropTypes.node,

  // A React element.
  optionalElement: PropTypes.element,

Reply:
Okay, I'm not sure what's the best practice for us then between `node` and `element`. But if it works, great.

Reply:
I changed to node.